plugin/trace/setup.go

```diff
@@ -100,6 +101,39 @@ func traceParse(c *caddy.Controller) (*trace, error) {
 				if tr.datadogAnalyticsRate > 1 || tr.datadogAnalyticsRate < 0 {
 					return nil, fmt.Errorf("datadog analytics rate must be between 0 and 1, '%f' is not supported", tr.datadogAnalyticsRate)
 				}
+			case "zipkin_max_backlog_size":
+				args := c.RemainingArgs()
+				if len(args) > 1 {
+					return nil, c.ArgErr()
+				}
+				if len(args) == 1 {
+					tr.zipkinMaxBacklogSize, err = strconv.Atoi(args[0])
+				}
```

Comment:
The new options accept no argument, in which case they do nothing. This isn't bad, but it's kinda weird, and not likely to be an intentional configuration on the user's part.  Suggest requiring a single argument...

```suggestion
				if len(args) != 1 {
					return nil, c.ArgErr()
				}
				tr.zipkinMaxBacklogSize, err = strconv.Atoi(args[0])
				
```

Reply:
yea, accepting no argument makes no sense. I fixed it as you suggested